content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: blog
+title: 'Kubernetes v1.30: TBA'
+date: 2024-04-17
+slug: kubernetes-v1-30-release
+---
+
+**Authors:** [Kubernetes v1.30 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.30/release_team.md)
+
+**Editors:** Amit Dsouza, Frederick Kautz, Kristin Martin, Abigail McCarthy, Natali Vlatko
+
+Announcing the release of Kubernetes v1.30: Uwubernetes, the cutest release!
+
+Similar to previous releases, the release of Kubernetes v1.30 introduces new stable, beta, and alpha
+features. The consistent delivery of top-notch releases underscores the strength of our development
+cycle and the vibrant support from our community.
+
+This release consists of 45 enhancements. Of those enhancements, 17 have graduated to Stable, 18 are
+entering Beta, and 10 have graduated to Alpha.
+
+## Release theme and logo
+
+(logo placeholder-pending high res file and resize)
+
+Kubernetes v1.30 makes your clusters cuter!
+
+Kubernetes is built and released by thousands of people from all over the world and all walks of
+life. Most contributors are not being paid to do this; we build it for fun, to solve a problem, to
+learn something, or for the simple love of the community. Many of us found our homes, our friends,
+and our careers here. The Release Team is honored to be a part of the continued growth of
+Kubernetes. 
+
+For the people who built it, for the people who release it, and for the furries who keep all of our
+clusters online, we present to you Kubernetes v1.30: Uwubernetes, the cutest release to date. The
+name is a portmanteau of “kubernetes” and “UwU,” an emoticon used to indicate happiness or cuteness.
+We’ve found joy here, but we’ve also brought joy from our outside lives that helps to make this
+community as weird and wonderful and welcoming as it is. We’re so happy to share our work with you. 
+
+UwU :heart:
```

Comment:
```suggestion
UwU ♥️
```

(unless we want the literal text `:heart:`)

Reply:
Kat's intro shows a heart emoji, this [chart](https://gist.github.com/rxaviers/7360908) shows `:heart:` to be the correct markdown entity.

Reply:
Okay, it seems GH supports emojis, while our site doesn't. Will change to unicode.

Reply:
GitHub uses [GitHub flavored Markdown](https://github.github.com/gfm/); we use [CommonMark](https://commonmark.org/) (with a few Hugo extensions).